setup.py

```diff
@@ -27,33 +27,38 @@
 import os
 import sys
 
-from datetime import datetime
 from setuptools import find_packages
 from setuptools import setup
 from setuptools.dist import Distribution
 from setuptools import Extension
 
+BASE_DIR = os.path.dirname(os.path.abspath(__file__))
+
+
+def get_last_commit_time() -> str:
+    from git import Repo
```

Comment:
okay to have this un-decalred dependency here since this should only be called using `--nightly` flag from the release script?

Reply:
Yes, that was my intention. It's only to build the nightlies that we need GitPython.